pkg/capabilities/consensus/ocr3/capability_test.go

```diff
@@ -20,6 +23,15 @@ import (
 const workflowTestID = "consensus-workflow-test-id-1"
 const workflowExecutionTestID = "consensus-workflow-execution-test-id-1"
 
+var transformJSON = cmp.FilterValues(func(x, y []byte) bool {
```

Comment:
I ended up copy / pasting this multiple times... where should i put this if i want to have test helpers?

Reply:
If these are test helpers my preference is to leave them copied since I don't think it harms much.

However there is a test utils package in common IIRC, so you could put it there too.

Reply:
yeah they're test helpers. Okay, I'll keep copy pasting them for now then.